### PR TITLE
feat: specify query and search params

### PR DESCRIPTION
**Pull request by mikaa123**
This one refers to #2. It makes it possible to a `query` and a `params` parameter to `algoliaSitemap`.

### CODE REVIEW

**Comment by mikaa123** [comment]
great idea!

**Comment by mikaa123** [comment]
@Haroenv @vvo if you guys are ok, let's move forward with this :)

File: index.js [hunk 1]
```diff
@@ -62,7 +68,7 @@ function init({ algoliaConfig, sitemapLoc, outputFolder, hitToParams }) {
     });
   };
 
-  return index.browse().then(aggregator);
+  return index.browse(params).then(aggregator);
```
Comment (Haroenv) [inline index.js]: What happens if it's undefined, does it still work like before?

Comment (mikaa123) [inline index.js]: yep, when `undefined` it's the same behavior as before (the default one)

**Comment by Haroenv** [comment]
will release a new version later this evening 😄

**Pull request merged**